content/ru/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -0,0 +1,53 @@
+
+{{<note>}}
+* Когда вы удаляете объект, Kubernetes добавляет метку времени удаления для этого объекта, а затем сразу же начинает ограничивать изменения в поле .metadata.finalizers для объекта, который сейчас ожидает удаления. Вы можете удалить существующие финализаторы (удалив запись из списка финализаторов), но не можете добавить новый финализатор. Вы также не можете изменить значение deletionTimestamp для объекта после его установки.
+
+* После запроса на удаление вы не сможете воскресить этот объект. Единственный способ — удалить его и создать новый аналогичный объект.
+{{</note>}}
+
+## Ссылки на владельцев, метки и финализаторы
+
+Как и метки, [ссылки на владельца](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/) описывают отношения между объектами в Kubernetes, но используются для другой цели. Когда контроллер управляет такими объектами, как модули, он использует метки для отслеживания изменений в группах связанных объектов. Например, когда задание создает один или несколько модулей Pod, контроллер заданий применяет метки к этим модулям и отслеживает изменения во всех модулях в кластере с той же меткой.
+
+Контроллер заданий также добавляет ссылки на владельцев этих модулей, указывая на задание, создавшее эти модули. Если вы удалите задание во время работы этих модулей, Kubernetes будет использовать ссылки на владельцев (а не метки), чтобы определить, какие модули в кластере нуждаются в очистке.
+
+Kubernetes также обрабатывает финализаторы, когда идентифицирует ссылки на владельца ресурса, предназначенного для удаления.
+
+В некоторых ситуациях финализаторы могут блокировать удаление зависимых объектов, в результате чего целевой объект-владелец может оставаться дольше, чем ожидалось, без полного удаления. В таких ситуациях вам следует проверить финализаторы и ссылки на владельца целевого владельца и зависимых объектов, чтобы устранить причину.
+
+{{<note>}}
+В случаях, когда объекты застревают в состоянии удаления, не удаляйте финализаторы вручную, чтобы продолжить удаление. Финализаторы обычно добавляются к ресурсам не просто так, поэтому их принудительное удаление может привести к проблемам в вашем кластере. Это следует делать только тогда, когда цель финализатора понятна и достигается другим способом (например, очисткой какого-либо зависимого объекта вручную).
+{{</note>}}
+
+## {{% heading "Что дальше" %}}
+
+Прочтите статью [«Использование финализаторов для управления удалением»](https://kubernetes.io/blog/2021/05/14/using-finalizers-to-control-deletion/) в блоге Kubernetes.
```

Comment:
Since it's not translated into Russian, it's better to leave the English title here, so users would expect the text in English.